task_sdk/src/airflow/sdk/definitions/connection.py

```diff
@@ -58,29 +57,18 @@ def get_uri(self): ...
 
     def get_hook(self): ...
 
-    @property
-    def extra_dejson(self, nested: bool = False) -> dict:
-        """
-        Deserialize extra property to JSON.
-
-        :param nested: Determines whether nested structures are also deserialized into JSON (default False).
-        """
-        extra_json = {}
-
+    def _get_extra_dejson(self) -> dict:
```

Comment:
Don't see a reason for this internal method

Reply:
let's just keep the logic in `extra_dejson`

Reply:
Cool, will do

Reply:
Doesnt make sense anymore, earlier it did due to nested parsing, masking etc.